README.md

```diff
@@ -1,24 +1,216 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <img src="logo.PNG" alt="logo" width="auto"  height="auto" />
+  <br/>
+  </div>
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 [Budget App] <a name="about-project"></a>
 
-* Database creation
+**[Buget App]** is a classic example of a blog website. It is a functional website that show the list of Categories and associated operations to each categories.
 
-* Database initialization
+**[Video Description](https://drive.google.com/file/d/1R-GI7cmCG04wLd3bvoDH8fmlJsp5wSMx/view?usp=drive_link)**
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+  <summary>Server</summary>
+   <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Splash_Screen]**
+- **[Categories_Screen]**
+- **[Operations_Screen]**
+- **[Users_Table]**
+- **[Category_Table]**
+- **[Operation_Table]**
+- **[Operation_Category_Table]**
+- **[Rspec_Testing]**
+- **[Add_New_Category]**
+- **[Add_New_Transaction]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://budgetapp-fnyd.onrender.com)
```

Comment:
- [ ] As per project requirements, the deployed link should work as expected. At the moment, I am unable to SignUp. When I fill the form and click on Sign Up, nothing happens. Please check and ensure that the Live link is working.
![image](https://github.com/Outkarm/Budget-app/assets/62849872/a4b16849-62d2-4c60-9789-ab1dddc2295b)

Reply:
just refresh the page